### ISSUE TITLE
Help required for using the code

### ISSUE THREAD

**Issue by senth27** [dao-duc-tung/Qml-VTK-Python#3]
Hi Dao Duc Tung,

I am trying out your python code to use the VTK window as QML Fbo object.

I am using Anaconda with Python 3.7, and installed PySide 5.11.2 and downloaded the vtk 9.0.1 wheel file from vtk.org site and installed on top of the anaconda.

The problem is the vtk install does not contain the vtkExternalOpenGLRenderWindow and it gives the loading error. Following command on the python interpreter,
**from vtk import vtkExternalOpenGLRenderWindow**      
gives the loading error.

But importing vtkGenericOpenGLRenderWindow works ok.

I changed the python code by replacing the vtkExternalOpenGLRenderWindow  with vtkGenericOpenGLRenderWindow , but the viewer does not show up.

Can you help me, how can we get the wheel file for VTK, which is built with vtkRenderingExternal option enabled. I am surprised to see the wheel files put on the https://vtk.org/download/ site, does not enable this option vtkRenderingExternal by default.

I am glad to see your location as Singapore, same as mine. 

Hope you can help me on this regard.

Thanks & Best Wishes,
Senthil

**Comment by dao-duc-tung** [comment]
Hi Senthil,

I'm glad you ask because building VTK again by yourself might take a lot of time and might need some tricks.

I have uploaded the VTK wheel file built with vtkRenderingExternal enabled here for you:
https://drive.google.com/file/d/1X9nJQ9KeA2VSc762d4Fu07sm7oJBs5t2/view?usp=sharing

After you download it, I will delete it from my GG drive so please keep it carefully 😄 

Anw, shout out from Singapore 🇸🇬 !
Tung

**Comment by dao-duc-tung** [comment]
Hi Senthil,

Thank you for reaching my repo.

First thing is your PySide2 version is not compatible. You should upgrade your PySide2 version to 5.13.x.

Second thing is vtk 9.0.1 is not compatible. Please use VTK version 8.1.x or 8.2.x. You can install vtk using `pip install vtk==8.1.2`

I have updated the code to use `vtkGenericOpenGLRenderWindow` by default. Please check the Compatibility and Installation sections in README file.

Btw, shout out from Singapore 🇸🇬 !
Tung

**Comment by senth27** [comment]
Hi Tung,

Thank you very much for your kind response and quick help.

I took your latest code from github repository and installed VTK 8.1.2. I tried with PySide2 5.13.2 and 5.15.2 and it runs perfectly. I used the downloaded wheel files from pypi.org/project site for VTK and PySide2 with shiboken2 whl files.

Thank you very much and your kind help is highly appreciated!

I must treat you for a dinner, after the lockdown is over!!!

Best Wishes,
Senthil
